src/GHC/RTS/Events.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE CPP,BangPatterns #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE MultiWayIf #-}
+{-# OPTIONS_GHC -fsimpl-tick-factor=150 #-}
```

Comment:
If we delete this line, GHC fails to compile this module like this:
```
ghc: panic! (the 'impossible' happened)
  (GHC version 8.0.2 for x86_64-apple-darwin):
        Simplifier ticks exhausted
  When trying UnfoldingDone $fNumInt_$c+
  To increase the limit, use -fsimpl-tick-factor=N (default 100)
  If you need to do this, let GHC HQ know, and what factor you needed
  To see detailed counts use -ddump-simpl-stats
  Total ticks: 196000
```

See [-fsimpl-tick-factor](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/using-optimisation.html?highlight=#ghc-flag--fsimpl-tick-factor) for the explanation of this flag. I've increased it from 100 to 150 to avoid this panic. https://ghc.haskell.org/trac/ghc/ticket/11263 is a similar issue.

I haven't tracked down the root cause but I think it's benign as we have neither custom `RULES` nor `INLINE`s in the code.